méli-mélo/2021-05-steps/a11y-1-steps.json

```
@@ -0,0 +1,651 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+    "@type": [
+        "earl:Assertion",
+        "acr:AssessmentReport"
+    ],
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/m%C3%A9li-m%C3%A9lo-demos/2024-09-kejimkujik/2021-05-steps",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Assess the GCWeb meli-melo steps page according to Standard on Web Accessibility (SWA)",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "steps"

```

Must be a valid CSS selector, there is no element `<steps>`.

```suggestion
            "@value": ".lst-stps"
```